src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml

```diff
@@ -62,13 +84,6 @@ Behavior:
     - name: video_camera_2
       description: Two IR video cameras (AAK CA20 600TVL 2.8MM) are used to monitor the experiments from different angles of interest, allowing for offline analysis of body movements, pupillometry, and other behavioral data if necessary. Camera 2 is a zoomed-in view of the pupil of the mouse.
       unit: Frames
-  Devices:
```

Comment:
Was this removal intended?

Reply:
Yes, you'll notice that these fields are moved up to line 55